src/main/java/domain/constant/GameResult.java

```diff
@@ -1,5 +1,17 @@
 package domain.constant;
 
 public enum GameResult {
-    WIN, LOSE
+    WIN(1),
+    LOSE(-1),
+    WIN_BY_BLACKJACK(1.5);
+
```

Comment:
새로운 enum이 추가될 때를 생각해서 아래처럼 쓰기도 합니다!
```suggestion
    LOSE(-1),
    WIN_BY_BLACKJACK(1.5),
    ;
```

Reply:
추가를 감안해서 뒤에 ','를 붙여서 사용하는 방법이 되는 줄 몰랐었습니다!
새로운 방법 알려주셔서 감사합니다 😊